--- origsrc/geany-0.17/plugins/Makefile.am	2009-02-25 19:25:38.000000000 -0600
+++ src/geany-0.17/plugins/Makefile.am	2009-06-08 18:47:04.125062300 -0500
@@ -20,13 +20,13 @@ geanyfunctions.h: genapi.py ../src/plugi
 all: geanyfunctions.h
 
 
-demoplugin_la_LDFLAGS    = -module -avoid-version
-classbuilder_la_LDFLAGS  = -module -avoid-version
-htmlchars_la_LDFLAGS     = -module -avoid-version
-export_la_LDFLAGS        = -module -avoid-version
-saveactions_la_LDFLAGS   = -module -avoid-version
-filebrowser_la_LDFLAGS   = -module -avoid-version
-splitwindow_la_LDFLAGS   = -module -avoid-version
+demoplugin_la_LDFLAGS    = -module -avoid-version -no-undefined
+classbuilder_la_LDFLAGS  = -module -avoid-version -no-undefined
+htmlchars_la_LDFLAGS     = -module -avoid-version -no-undefined
+export_la_LDFLAGS        = -module -avoid-version -no-undefined
+saveactions_la_LDFLAGS   = -module -avoid-version -no-undefined
+filebrowser_la_LDFLAGS   = -module -avoid-version -no-undefined
+splitwindow_la_LDFLAGS   = -module -avoid-version -no-undefined
 
 if PLUGINS
 
