--- origsrc/geany-1.24.1/data/filetype_extensions.conf	2014-04-16 10:20:12.000000000 -0500
+++ src/geany-1.24.1/data/filetype_extensions.conf	2014-04-24 11:09:19.859422800 -0500
@@ -56,7 +56,7 @@ R=*.R;*.r;
 Rust=*.rs;
 Ruby=*.rb;*.rhtml;*.ruby;*.gemspec;Gemfile;rakefile;Rakefile;
 Scala=*.scala;*.scl;
-Sh=*.sh;configure;configure.in;configure.in.in;configure.ac;*.ksh;*.mksh;*.zsh;*.ash;*.bash;*.m4;PKGBUILD;*profile;
+Sh=*.sh;configure;configure.in;configure.in.in;configure.ac;*.ksh;*.mksh;*.zsh;*.ash;*.bash;*.m4;PKGBUILD;*profile;*.cygport;*.cygpart;*.cygclass;
 SQL=*.sql;
 Tcl=*.tcl;*.tk;*.wish;
 Txt2tags=*.t2t;
