--- origsrc/geany-0.14/plugins/Makefile.am	2008-04-09 11:21:59.000000000 -0500
+++ src/geany-0.14/plugins/Makefile.am	2008-04-24 13:32:19.843750000 -0500
@@ -9,13 +9,13 @@
 plugins_includedir = $(includedir)/geany/
 plugins_include_HEADERS = pluginmacros.h
 
-demoplugin_la_LDFLAGS    = -module -avoid-version
-classbuilder_la_LDFLAGS  = -module -avoid-version
-htmlchars_la_LDFLAGS     = -module -avoid-version
-export_la_LDFLAGS        = -module -avoid-version
-vcdiff_la_LDFLAGS        = -module -avoid-version
-autosave_la_LDFLAGS      = -module -avoid-version
-filebrowser_la_LDFLAGS   = -module -avoid-version
+demoplugin_la_LDFLAGS    = -module -avoid-version -no-undefined
+classbuilder_la_LDFLAGS  = -module -avoid-version -no-undefined
+htmlchars_la_LDFLAGS     = -module -avoid-version -no-undefined
+export_la_LDFLAGS        = -module -avoid-version -no-undefined
+vcdiff_la_LDFLAGS        = -module -avoid-version -no-undefined
+autosave_la_LDFLAGS      = -module -avoid-version -no-undefined
+filebrowser_la_LDFLAGS   = -module -avoid-version -no-undefined
 
 if PLUGINS
 
@@ -40,14 +40,13 @@
 autosave_la_SOURCES      = autosave.c
 filebrowser_la_SOURCES   = filebrowser.c
 
-# we don't need to link against GTK libraries (or do we?)
-#~ demoplugin_la_LIBADD    = $(GTK_LIBS)
-#~ classbuilder_la_LIBADD  = $(GTK_LIBS)
-#~ htmlchars_la_LIBADD     = $(GTK_LIBS)
-#~ export_la_LIBADD        = $(GTK_LIBS)
-#~ vcdiff_la_LIBADD        = $(GTK_LIBS)
-#~ autosave_la_LIBADD      = $(GTK_LIBS)
-#~ filebrowser_la_LIBADD   = $(GTK_LIBS)
+demoplugin_la_LIBADD    = $(GTK_LIBS)
+classbuilder_la_LIBADD  = $(GTK_LIBS)
+htmlchars_la_LIBADD     = $(GTK_LIBS)
+export_la_LIBADD        = $(GTK_LIBS)
+vcdiff_la_LIBADD        = $(GTK_LIBS)
+autosave_la_LIBADD      = $(GTK_LIBS)
+filebrowser_la_LIBADD   = $(GTK_LIBS)
 
 endif # PLUGINS
 
--- origsrc/geany-0.14/src/plugins.c	2008-04-17 12:48:12.000000000 -0500
+++ src/geany-0.14/src/plugins.c	2008-04-24 13:24:41.593750000 -0500
@@ -57,7 +57,7 @@
 #include "navqueue.h"
 
 
-#ifdef G_OS_WIN32
+#ifdef G_PLATFORM_WIN32
 # define PLUGIN_EXT "dll"
 #else
 # define PLUGIN_EXT "so"
